tasks/system_probe.py

```diff
@@ -669,6 +669,85 @@ def test(
         raise Exit(code=1, message="system-probe tests failed")
 
 
+@task(
+    help={
+        "package": "The package to test. REQUIRED ",
+        "skip_object_files": "Skip rebuilding the object files.",
+        "run": "The name of the test to run. REQUIRED",
+    }
+)
+def test_debug(
+    ctx,
+    package,
```

Comment:
What happens if package is not specified?

Reply:
Since it does not have a default value, `invoke` makes it a required argument, and will print error stating that.